src/aleph/vm/pool.py

```diff
@@ -123,7 +123,7 @@ async def create_a_vm(
                 await execution.start()
 
                 # Start VM and snapshots automatically
-                if execution.persistent:
+                if execution.persistent and not execution.is_confidential:
```

Comment:
Can you add a comment to explain why confidential VMs are excluded ?

Reply:
We need to wait for the user session files (session.b64 and godh.b64 files) before start the instance using systemd, this is why I'm creating the new endpoint, to receive that files and start the systemd service once we have it.